contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -657,68 +737,82 @@ def search_observables_exist_validate(self):
         # Found everything
         return self
 
-    @model_validator(mode='after')
-    def ensurePresenceOfRequiredTests(self):
```

Comment:
Can we remove this? The check here is identical to what is being performed in the `tests_validate` func

Reply:
Given the presence of the `skip_manual_tests` function, yes.